netbout-client/src/main/java/com/netbout/mock/MkBase.java

```diff
@@ -122,5 +110,26 @@ public Bout randomBout() throws IOException {
         );
         return bout;
     }
+    /**
+     * Create alias.
+     * @param name Name of the alias
+     * @return Alias
+     * @throws IOException If fails
+     */
+    public Alias alias(final String name) throws IOException {
```

Comment:
@erimerturk but is has a randomly generated component - which also occurs in the `randomAlias` method. Why do we need a new method?